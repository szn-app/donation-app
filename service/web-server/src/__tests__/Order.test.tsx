import { expect, test } from "vitest";
import { render } from "@testing-library/react";
import Order from "../components/Order";

test("snapshot with nothing in order", () => {
  const { asFragment } = render(<Order />);
  expect(asFragment()).toMatchSnapshot();
});
