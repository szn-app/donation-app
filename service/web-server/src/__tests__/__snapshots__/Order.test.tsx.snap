// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`snapshot with nothing in order 1`] = `
<DocumentFragment>
  <div
    class="order"
  >
    <h2>
      Create Order
    </h2>
    <form
      action=""
    >
      <div>
        <div>
          <label
            for="pizza-type"
          >
            Pizza Type
          </label>
          <select
            name="pizza-type"
          >
            <option
              value="pepperoni"
            >
              pizza pepperoni
            </option>
            <option
              value="hawaiian"
            >
              pizaa hawaiian
            </option>
            <option
              value="big_meat"
            />
          </select>
        </div>
        <button />
      </div>
    </form>
  </div>
</DocumentFragment>
`;
